### ISSUE TITLE
missin MCLR pin on my board

### ISSUE THREAD

**Issue by g14n7uca** [Dfinitski/N7DDC-ATU-100-mini-and-extended-boards#96]
Purchased last week
The MCLR pin in my board is not soldered so oly 4 pins came out to connect it to pic programmer.
Suppose I have to solder one pin and then connect it to pic it 3...isn't it?
Thanks

**Comment by pimol** [comment]
https://www.dropbox.com/s/jcyapzbwugt5avn/image001%20%281%29.png?dl=0

Pietro I2OIM

Il gio 14 lug 2022, 06:25 g14n7uca ***@***.***> ha scritto:

**Comment by g14n7uca** [comment]
Thanks pietro,
I meant, miss the phisical pin on the pcboard

![atu100](https://user-images.githubusercontent.com/109263753/178920534-8206d1e0-c9af-47d7-bec7-5523a0bd02f8.jpg)

**Comment by pimol** [comment]
MCLR has to be connected with a 4.7k to VCC, as indicated. Not important
that there is no pin on the  board.
Good luck

Pietro I2OIM

Il gio 14 lug 2022, 08:57 g14n7uca ***@***.***> ha scritto:

**Comment by pimol** [comment]
Sorry, I'm not clear.
You have to put the pin in the board.


Pietro

Il gio 14 lug 2022, 09:03 Pietro Molina ***@***.***> ha
scritto:

**Comment by g14n7uca** [comment]
...and connect it to the pic board I suppose, that was the question.

**Comment by pimol** [comment]
Yes, you have to complete the connector. I had the same problem.

Pietro I2OIM

Il giorno gio 14 lug 2022 alle ore 09:10 g14n7uca ***@***.***>
ha scritto:
